config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml

```diff
@@ -45,6 +45,24 @@ spec:
                     - s390x
                     - ppc64le
 
+   # callHome is a statefulSet with ibm-block-csi-call-home container
+#  callHome:
+#    repository: stg-artifactory.xiv.ibm.com:5030/ibm-block-csi-call-home-amd64
```

Comment:
please remove this

Reply:
what is the "planned public destination"?

Reply:
whatever we decide.
what do you think about `ibmcom/ibm-block-csi-call-home`?

Reply:
done